charts/datadog-operator/values.yaml

```diff
@@ -106,6 +106,13 @@ datadog-crds:
     datadogMetrics: true
     # datadog-crds.crds.datadogMonitors -- Set to true to deploy the DatadogMonitors CRD
     datadogMonitors: true
+  migration:
+    datadogAgents:
+      version: "v1alpha1"
+
+migration:
+  webhook:
+    enabled: false
```

Comment:
maybe you can aggregate this migration in one block

```yaml
migration:
  datadogAgents:
    version: "v1alpha1"
  webhook:
    enabled: false 
```

also it can be done maybe in another way

```yaml
supportedDatadogAgentVersions:
  v1alpha1: true
  v2alpha1: true
```

Reply:
I agree, as is, I would be better off aggregating the sections. However helm does not let users use `-` in variable names, so I can't have options under the `datadog-crds` option be used in the logic of the datadog-operator chart.
However, I realized that we will be adding more webhook backed features in the future so it makes sense to isolate that section in something more appropriate such as:

```yaml
webhook:
  conversion:
    enabled: false
```